src/ImageRunModal.jsx

```diff
@@ -693,8 +703,16 @@
                     <TextInput id='run-image-dialog-name'
                            className="image-name"
                            placeholder={_("Container name")}
+                           validated={dialogValues.validationFailed.containerName ? "error" : "default"}
+                           onFocus={() => utils.validationClear(dialogValues.validationFailed, "containerName", (value) => this.onValueChanged("validationFailed", value))}
+                           onBlur={async () => {
+                               const delta = await this.validateContainerName(dialogValues.containerName);
+                               if (delta)
+                                   this.onValueChanged("validationFailed", { ...dialogValues.validationFailed, containerName: delta });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230725-122305-68cb803a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)